src/jsTest/kotlin/process/phonemes/PhonemesMappingTest.kt

```diff
@@ -73,8 +75,16 @@ class PhonemesMappingTest {
 
     @Test
     fun testSortLength() {
-        val note = createNote("ef")
+        val note = createNote("sh")
         val actual = note.replacePhonemes(request).phoneme
-        assertEquals("EF", actual)
+        assertEquals("SH", actual)
+    }
+
+    @Test
+    fun testSortMultiLength() {
+        println(request.map)
+        val note = createNote("effff d c a")
+        val actual = note.replacePhonemes(request).phoneme
+        assertEquals("EF DC' A", actual)
```

Comment:
Test currently fails since `replacePhonemes()` finds the `effff d` key at the beginning of the note before the `d c a` key at the end of the note